UpdateDependencies.ps1

```diff
@@ -16,7 +16,7 @@ param(
     [string]$GitHubProject='corefx',
     [string]$GitHubUpstreamBranch='master',
     # a semi-colon delimited list of GitHub users to notify on the PR
-    [string]$GitHubPullRequestNotifications='')
+    [string]$GitHubPullRequestNotifications='stephentoub;ericstj;ellismg;weshaggard;joshfree')
```

Comment:
My plan was to keep the default empty and put the actual list of people in the "subscription" information (where things like dotnet-bot's email is specified). 
https://github.com/eerhardt/versions/blob/master/Subscriptions.json

The advantage here is that to change the list we don't have to go through a full CoreFX PR and test run.

Another small advantage is when I (or someone else) modifies the script and wants to run it locally, by default no one will be notified.

It doesn't make too much difference either way I guess. But that was my reasoning for putting it at the "call site".